src/watertap_contrib/reflo/solar_models/surrogate/flat_plate/data/pysam_run_flat_plate.py

```diff
@@ -0,0 +1,515 @@
+import json
+from os.path import join, dirname
+from math import floor, ceil, isnan
+import numpy as np
+import pandas as pd
+import time
+import multiprocessing
+from itertools import product
+import matplotlib.pyplot as plt
+import PySAM.Swh as swh
+from watertap_contrib.reflo.solar_models.surrogate.flat_plate.data.pyssc import (
+    PySSC,
+)
+
+
```

Comment:
More comments throughout this file would be helpful. As of now, I don't think we have as many comments as we might like.